From: Sven Hartge <sven@svenhartge.de>
Date: Tue, 10 Apr 2012 21:42:13 +0200
Subject: [PATCH] Output some progress during CUSUM and RANDOM EXCURSION test

Those tests takes enough time on mips to outbound the allowed 5 minutes of
inactivity. So let's output a little more progress messages as a workaround.
---
 nist/packtest.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/nist/packtest.c b/nist/packtest.c
index 9439607..879cd73 100644
--- a/nist/packtest.c
+++ b/nist/packtest.c
@@ -557,6 +557,7 @@ PackTestL (int *ARRAY, int ArraySize, char *C)
   for (i = 4; i < ArraySize; i = (i << 1))
     {
       int inter;
+      fprintf (stderr, "\t\t   ArraySize: %d\n", i);
       inter = MOD (ARRAY[ArraySize - i], ArraySize - i);
       if (failure >= 8)
 	{
@@ -600,6 +601,7 @@ PackTestL (int *ARRAY, int ArraySize, char *C)
   for (i = 0; i < 8; i++)
     {
       int index;
+      fprintf (stderr, "\t\t      Slice number: %d\n", i);
       index = (ArraySize / 8) * i;
       if (ArraySize > 262144)
 	index = index + (MOD (ARRAY[index], (ArraySize / 8) - 32768));
-- 
